index.html

```diff
@@ -0,0 +1,196 @@
+              >Has to stop school and save up more money. Works in the Civilian
+              Conservation Corps, helping starving Americans. "I saw how food
+              changed them", he said. "All of this left scars on me."
+            </span>
+          </li>
+          <li>
+            <span class="date">1937</span> -
+            <span class="detail"
+              >Finishes university and takes a job in the US Forestry Service
+            </span>
+          </li>
+          <li>
+            <span class="date">1938</span> -
+            <span class="detail"
+              >Marries wife of 69 years Margret Gibson. Gets laid off due to
+              budget cuts. Inspired by Elvin Charles Stakman, he returns to
+              school study under Stakman, who teaches him about breeding
+              pest-resistent plants.
+            </span>
+          </li>
+          <li>
+            <span class="date">1941</span> -
+            <span class="detail"
+              >Tries to enroll in the military after the Pearl Harbor attack,
+              but is rejected. Instead, the military asked his lab to work on
+              waterproof glue, DDT to control malaria, disinfectants, and other
+              applied science.
+            </span>
+          </li>
+          <li><span class="date">1942</span> - <span class="detail"></span></li>
+          <li>
+            <span class="date"></span
+            ><span class="detail"
+              >Receives a Ph.D. in Genetics and Plant Pathology
+            </span>
+          </li>
+          <li>
+            <span class="date">1944</span> -
+            <span class="detail"
+              >Rejects a 100% salary increase from Dupont, leaves behind his
+              pregnant wife, and flies to Mexico to head a new plant pathology
+              program. Over the next 16 years, his team breeds 6,000 different
+              strains of disease resistent wheat - including different varieties
+              for each major climate on Earth.
+            </span>
+          </li>
+          <li>
+            <span class="date">1945</span> -
+            <span class="detail"
+              >Discovers a way to grown wheat twice each season, doubling wheat
+              yields
+            </span>
+          </li>
+
+            <span class="date">1953</span>
+            <span class="date">1953</span>
+            <span class="detail"
+              >crosses a short, sturdy dwarf breed ofwheat with a high-yeidling
+              American breed, creating a strain that responds well to
+              fertilizer. It goes on to provide 95% of Mexico's wheat</span
+            >
+          </li>
+          <li>
+            <span class="date">1962</span> -
+            <span class="detail"
+              >Visits Delhi andrings his high-yielding strains of wheat to the
+              Indian subcontinent in time to help mitigate mass starvation due
+              to a rapidly expanding population
```

Comment:
`p` tag

Reply:
> `p` tag

You did great for the first review about the use of `p` tag to ensure the accessibility 👏 . You also used polite language to suggest and gave a brief explanation `why`. But i suggest that the same review approach is used here. This will make the inline comment more meaningful to the project contributor(s) ☺️

Reply:
Kindly add a `p` tag in place of `span` tags. As mentioned previously `p` tags enable differentiation of the text levels optimizing SEO :computer: